CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- [Add client info to nrepl clone request](https://github.com/BetterThanTomorrow/calva/pull/2774)
```

Comment:
We want this to link to an issue which has the problem description. So that users curious about a change quickly can find out why it was made.

Reply:
Done!